examples/odata-cli/src/activities.ts

```diff
@@ -12,7 +12,6 @@ import { logger } from './types';
 
 /**
  * Execute a sequence of test calls using the given provider.
- *
```

Comment:
Do we really need to delete the newline between jsdoc description and parameters? This creates a lot of noise on this PR and personally I don't think it increases the readability of the jsdoc comments.

Reply:
@Klaus-Keller Yeah it is a lot of noise. I've adjusted the eslint config for the new version to keep the old behaviour for the newline after the jsdoc description
https://github.com/SAP/open-ux-tools/pull/1080/commits/4cc4eb32377ef5c056defa06f49f91dbcc369b26 and https://github.com/SAP/open-ux-tools/pull/1080/commits/29add5017080ea6a1f2148d6a6d671bf0b56019f

Reply:
Thanks @devinea!